test/conformance/gatewayapi/gateway_conformance_test.go

```diff
@@ -29,32 +34,45 @@ import (
 )
 
 func TestGatewayConformance(t *testing.T) {
+	log.SetLogger(logrusr.New(logrus.StandardLogger()))
+
 	cfg, err := config.GetConfig()
 	require.NoError(t, err)
 
 	client, err := client.New(cfg, client.Options{})
 	require.NoError(t, err)
 
+	clientset, err := kubernetes.NewForConfig(cfg)
+	require.NoError(t, err)
+
 	require.NoError(t, v1alpha2.AddToScheme(client.Scheme()))
 	require.NoError(t, v1beta1.AddToScheme(client.Scheme()))
 
 	cSuite := suite.New(suite.Options{
-		Client:                     client,
+		Client: client,
+		// This clientset is needed in addition to the client only because
+		// controller-runtime client doesn't support non CRUD sub-resources yet (https://github.com/kubernetes-sigs/controller-runtime/issues/452).
+		Clientset:                  clientset,
 		GatewayClassName:           *flags.GatewayClassName,
 		Debug:                      *flags.ShowDebug,
 		CleanupBaseResources:       *flags.CleanupBaseResources,
 		EnableAllSupportedFeatures: true,
 		// Keep the list of skipped features in sync with
 		// test/scripts/run-gateway-conformance.sh.
 		SkipTests: []string{
-			// These are skipped because the tests check for the
-			// original request port in the returned Location
+			// Test adds multiple HTTP Listeners to the Gateway with
+			// distinct ports. This is not supported in Contour until
+			// multi-Listener support is added.
+			// See: https://github.com/projectcontour/contour/issues/4960
+			tests.GatewayObservedGenerationBump.ShortName,
```

Comment:
I've confirmed that this test will pass with #5160, will update that PR to remove it from the skipped tests